coinstacks/ethereum/ingester/src/workers/txid.ts

```diff
@@ -10,16 +10,18 @@ if (!INDEXER_WS_URL) throw new Error('INDEXER_WS_URL env var not set')
 
 const blockbook = new Blockbook({ httpURL: INDEXER_URL, wsURL: INDEXER_WS_URL })
 
+const msgLogger = logger.child({ namespace: ['workers', 'txid'], fn: 'onMessage' })
 const onMessage = (worker: Worker) => async (message: Message) => {
   const txid: string = message.getContent()
 
   try {
     const tx = await blockbook.getTransaction(txid)
+    msgLogger.debug({ txid, tx }, 'getTransaction')
```

Comment:
Also removed these debug log as they drown out everything as well (figured easy to add by dev if they really care)

Reply:
setting the LOG_LEVEL to "info" would fix that, or these can be made to be `trace` calls.  You're free to remove/add whatever logs you want. I think I added it because this same log was in many other files in the same place.

Reply:
I like the idea of trace. I can comb through with a few updates